library/core/src/fmt/mod.rs

```diff
@@ -94,16 +94,14 @@ pub struct Error;
 
 /// A collection of methods that are required to format a message into a stream.
 ///
-/// This trait is the type which this modules requires when formatting
-/// information. This is similar to the standard library's [`io::Write`] trait,
-/// but it is only intended for use in libcore.
-///
-/// This trait should generally not be implemented by consumers of the standard
-/// library. The [`write!`] macro accepts an instance of [`io::Write`], and the
-/// [`io::Write`] trait is favored over implementing this trait.
-///
-/// [`write!`]: ../../std/macro.write.html
-/// [`io::Write`]: ../../std/io/trait.Write.html
+/// This trait is used by this module for formatting information.
+/// It is a subset of the [`std::io::Write`] trait because it only works for
```

Comment:
It's not a subset though, right? There's no blanket impls either direction? (At least I couldn't quickly spot any)

Reply:
I meant it's a "feature subset". Is that not true?

Reply:
I did say "close to superset/subset" as a way of expressing their relationship of one having an almost complete Venn overlap of the other, but it's not exactly a mathematical subset so the term "subset" strictly speaking should not be used (it can inject confusion, as witnessed in this conversation :sweat_drops:).

Reply:
I changed it to this:

```diff
 /// This trait is used by this module for formatting information.
-/// It is a subset of the [`std::io::Write`] trait because it only works for
+/// Its capabilities are a subset of the [`std::io::Write`] trait because it only works for
 /// UTF-8–encoded data and it is not [flushable]. If you only want to accept Unicode
```